app/services/taxonomy/links_update.rb

```diff
@@ -0,0 +1,32 @@
+module Taxonomy
+  class LinksUpdate
```

Comment:
The "why" would be nice to have in this commit message, even though I can guess that this rename is a sensible preparation step for adding more responsibilities to this class, I think it's useful to make that explicit in the commit message.